pkg/keyring/keyring.go

```diff
@@ -57,49 +51,13 @@ func DefaultKeyringForOS() (krPkg.Keyring, error) {
 	case "darwin":
 		cfg.AllowedBackends = []krPkg.BackendType{krPkg.KeychainBackend}
 	case "linux":
-		cfg.AllowedBackends = []krPkg.BackendType{krPkg.PassBackend}
```

Comment:
Not specifying AllowedBackends means all backends are OK.

![image](https://user-images.githubusercontent.com/562343/109126818-fa45d600-774d-11eb-9143-a8b34565ab4a.png)

Reply:
Just beware of the words `all available`, as we saw with the keychain backend, this was not available because we were not building with CGO enabled, thus not satisfying the cgo build tag.

Reply:
I did manually testing on virtualbox+linux and a remote macOS instance to verify the keychain is working as expected.